mod_tariffs.asciidoc

```diff
@@ -149,7 +149,7 @@ After all, the CPO is the owner of the object, so it would be illogical if the e
 [[mod_tariffs_msp_get_request_parameters]]
 ====== Request Parameters
 
-The following parameters shall be provided as URL segments.
+The following parameters MUST be provided as URL segments.
```

Comment:
If we go for all caps, so RFC 2119, I guess it should be SHALL, as all three parameters are required. 
On the other hand, the parameters them selves already define if they are required, that was the reason I did not make it all caps. SHALL. Your idea on this?

Reply:
I admit I am a little bit confused on the difference between MUST, REQUIRED, and SHALL.  To me, they feel the same so why have multiple words to mean the same thing in RFC 2118 standard?  As long as you are consistent in using SHALL, that would be the best for OCPI.

I feel that it can't possibly hurt (judging from the comments I am see from myself and others) to be extra emphatic about the fact that all three parameters are required so I would feel that SHALL would be best.

Reply:
I agree SHALL is better. Can you change this?

Reply:
I believe I have made the changes.  And hopefully pushed them into the pull-request.  If you don't see it, let me know and I'll try again.  I also looked at the other two places where "shall" was being used and changed them to "SHALL".